### PR TITLE
Move migrated pages on top of old pages

### PR DESCRIPTION
**Pull request by kwalrath**
I also deleted most of the old example files, except for the ones from src/codelabs/dart-cheatsheet.md & src/samples/index.md. Those ones are under /examples/misc, not a page-specific directory, and it wasn't as obvious what was safe to remove. I can take care of that in a later PR.

Let's see what CI thinks of this.

### CODE REVIEW

**Comment by kwalrath** [comment]
I *think* this is ready, but since I'll be out tomorrow, maybe we should wait until Monday to merge it.

Staged:
https://kw-staging-dartlang-2.firebaseapp.com/guides/language/language-tour
https://kw-staging-dartlang-2.firebaseapp.com/guides/libraries/library-tour
https://kw-staging-dartlang-2.firebaseapp.com/codelabs
https://kw-staging-dartlang-2.firebaseapp.com/samples

**Review by domesticmouse — APPROVED** [review]

**Review by johnpryan — APPROVED** [review]

**Pull request merged**
